language/sentiment/mixed.txt

```diff
@@ -0,0 +1,20 @@
+I really wanted to love 'Bladerunner' but ultimately I couldn't get
```

Comment:
The "sentiment" dir refers to this tutorial, and I'd like to keep it named for what it is. Did you mean move the test files into a subdirectory: "sentiment/resources"? I can do that.